app/models/ncr/proposal_form.rb

```diff
@@ -92,12 +89,33 @@ def set_props_on(cart)
         vendor: self.vendor,
         not_to_exceed: self.not_to_exceed,
         building_number: self.building_number,
-        rwa_number: self.rwa_number,
         office: self.office
       )
+      case self.expense_type
+        when 'BA61'
+          # Hack to account for SimpleFormObject bugs
+          cart.set_props(emergency: self.emergency != false && self.emergency != "0")
```

Comment:
Is the intention to store the string, or an actual boolean? Anyway, I would pull this out to an `emergency_bool` method (or something).

http://stackoverflow.com/questions/8119970/string-true-and-false-to-boolean#comment32650264_21042466